### PR TITLE
[TSK-172] create buildSrc module and move versions

### PR DESCRIPTION
**Pull request by Hoocki**
finally

### CODE REVIEW

**Comment by adsukharev** [comment]
Resolve conflicts, tomorrow I will check

File: build.gradle.kts [hunk 1]
```diff
@@ -4,24 +4,12 @@ plugins {
 	id("io.freefair.lombok") version "8.4" apply false
 }
 
-group = "com.qpay"
-version = "0.0.1-SNAPSHOT"
```
Comment (adsukharev) [inline build.gradle.kts:8]: why did you remove group and version?

File: build.gradle.kts [hunk 2]
```diff
@@ -4,24 +4,12 @@ plugins {
...
-	set("webFlux", "3.2.2")
-	set("wireMock", "3.3.1")
-	set("jakartaValidation", "3.0.0")
-	set("kafka", "3.1.1")
-	set("zxingCore", "3.5.3")
-	set("zxingJavaSe", "3.5.3")
+allprojects {
+	repositories {
+		mavenCentral()
+	}
```
Comment (adsukharev) [inline build.gradle.kts]: you incorrectly rebased branch. Check previous changes or ask Kostya

File: libs/build.gradle.kts [hunk 3]
```diff
@@ -13,10 +13,8 @@ configurations {
     }
 }
 
-val jakartaValidation = rootProject.extra["jakartaValidation"]
-
 dependencies {
 
-    implementation("jakarta.validation:jakarta.validation-api:$jakartaValidation")
+    implementation("jakarta.validation:jakarta.validation-api:${Versions.JAKARTA_VALIDATION}")
```
Comment (adsukharev) [inline libs/build.gradle.kts:18]: As I said previously, use direct import

File: build.gradle.kts (hunk 1 above)
Comment (Hoocki) [reply build.gradle.kts:8]: i thought, it's optional and can be removed, but it would be good to know group of project and version, so i will return it

Comment (adsukharev) [reply build.gradle.kts:8]: I don't see that you returned it

**Review by adsukharev — APPROVED** [review]
